pylintrc

```diff
@@ -439,7 +439,9 @@ int-import-graph=
 known-standard-library=
 
 # Force import order to recognize a module as part of a third party library.
-known-third-party=enchant
+known-third-party=enchant,
+                  geoh5py,
+                  geoapps_utils
```

Comment:
for some reason necessary for pylint to recognize these as third parties
and stop complaining about the import order